infra/cifuzz/filestore/no_filestore/__init__.py

```diff
@@ -26,7 +26,7 @@ def upload_crashes(self, name, directory):
     """Noop implementation of upload_crashes."""
     logging.info('Not uploading crashes because no Filestore.')
 
-  def upload_corpus(self, name, directory):
+  def upload_corpus(self, name, directory, replace=False):
```

Comment:
Why make this change?

Reply:
It is defined with `replace` in `class BaseFilestore`

When I started my `GitlabFilestore`, I started by copying the no_filestore, and then I ended up with an error that `replace` was an unknown argument for `upload_corpus`

Reply:
ACK.
I'll merge the fix for this here: https://github.com/google/oss-fuzz/pull/7069